app/models/experimentation/notifications_experiment.rb

```diff
@@ -81,17 +84,25 @@ def monitor
       end
     end
 
+    def messages_report_key
+      if start_time > NEW_MESSAGE_REPORTING_DATE
+        NEW_MESSAGE_REPORTING_DATE
```

Comment:
Should this be `NEW_MESSAGE_REPORTING_KEY`?

Reply:
Fixed in [0729c29](https://github.com/simpledotorg/simple-server/pull/4110/commits/0729c29ae51489e55337bb33a724c575ea462807)